ubuntu-classic-23.04-amd64.json

```diff
@@ -0,0 +1,81 @@
+{
+    "type": "model",
+    "series": "16",
+    "authority-id": "canonical",
+    "brand-id": "canonical",
+    "model": "ubuntu-classic-23.04-amd64",
+    "architecture": "amd64",
+    "timestamp": "2023-03-19T12:00:00.0Z",
+    "base": "core22",
+    "grade": "signed",
+    "classic": "true",
+    "distribution": "ubuntu",
+    "snaps": [
+        {
+            "name": "pc",
+            "type": "gadget",
+            "default-channel": "23.04/stable",
```

Comment:
should the track name for the gadget be more explicit that this is for a classic / hybrid / whatever system? or is the fact that it has a xx.yy name enough?

Reply:
I would say the series version in the name is good enough and self-explanatory (to devs, since it's not something users will have contact with), but I'm open to propositions!

Reply:
are we implicitly saying that for 24.04 we will have a single gadget for iot and hybrid? that might not be possible

Reply:
No. In my eyes we will have a 24/ gadget and a 24.04/ one. We *always* reference Ubuntu series by XX.YY, never by XX because we have 2 different ones each year. So in my eyes a track that is just XX/ means core series. So we can still have a 24/ track for core and 24.04, then 24.10 etc. for classic.

Reply:
I fear that having tracks for pc gadget named XX.XX might confuse people and they could try to use them to build UC images. Having classic/hybrid as prefix would help with that problem.

Reply:
mmh, I wonder if it makes sense for us but is too subtle for our customers, there is nothing super obvious about the difference between 24 and 24.04